### ISSUE TITLE
Buttock pain

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#9869]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
PMID 17828787 : Buttock Pain on Sitting
Gluteal innervation is not dependent on the pudendal nerve and isolated buttock pain, even occurring while sitting,
cannot be considered to be due to pudendal neuralgia.
However, the combination of perineal neuralgia and buttock pain can be explained by a common, fairly proximal conflict underneath the piriformis muscle with a concomitant lesion of the posterior femoral cutaneous nerve or inferior gluteal nerve. Buttock pain can be related to trigger points or spasm of deep gluteal muscles: obturator internus and piriformis muscles, possibly due to reflex muscle
contractures secondary to pain or regional myofascial syndrome, extremely common in this context (reflecting regional
hypersensitivity)

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Pudendal neuralgia ORPHA:60039

**Your nano-attribution (ORCID)**
PMID 22110274 : Clinical presentation
It is commonly described as a superficial pain, burning sensation, numbness, or paresthesia in the gluteal, 
perineal, and/or genital areas[23]. It may be homolateral or bilateral, radiate to the pelvis and the thighs, and be associated with deep pelvic discomfort[33,34]. Pain may be worsened by sexual intercourse and initially reported as sciatic pain[33,34]. The epidemiology of pudendal neuralgia in the general population is unknown. The diagnosis is rarely considered except in highly focused Pelvic Floor Units or in specialized Urogynecologist practices. It is usually considered to be a rare entity, but it may be overdiagnosed due to the functional comorbidities associated with pudendal nerve dysfunction[33,34]. Recently, a multidisciplinary Committee reported that pudendal neuralgia may be simply diagnosed by default in the presence of pelvic, perineal, and **buttock pain** without evidence of organic disease at diagnostic workup[35]. Particularly controversial is its association with rectal pain, the presence 
of which requires differential diagnosis with chronic proctalgia

**Comment by pnrobinson** [comment]
Added as HP:0034805
